actors/user/src/main/java/org/sunbird/user/actors/IdentifierFreeUpActor.java

```diff
@@ -0,0 +1,106 @@
+package org.sunbird.user.actors;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.actor.core.BaseActor;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.util.Util;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@ActorConfig(
+        tasks = {"freeUpUserIdentity"},
+        asyncTasks = {}
+)
+public class IdentifierFreeUpActor extends BaseActor {
+
+    private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+    private Util.DbInfo usrDbInfo = Util.dbInfoMap.get(JsonKey.USER_DB);
+    private Map<String, Object> userDbMap = new HashMap<>();
```

Comment:
no need to initialise userDbMap here.

Reply:
Add java doc for class and all public method.

Reply:
done